commands/user_e2e_test.go

```diff
@@ -244,6 +244,26 @@ func (s *MmctlE2ETestSuite) TestCreateUserCmd() {
 		s.Equal(username, user.Username)
 		s.Equal(false, user.IsSystemAdmin())
 	})
+
+	s.RunForSystemAdminAndLocal("Should create new user with the email already verified only for admin or local mode", func(c client.Client) {
```

Comment:
Maybe we can add a check in the upper test case if the `user.EmailVerified` is `false`. Or add a new test with a regular user to ensure they cannot verify users at creation.